contracts/governance/Governor.sol

```diff
@@ -0,0 +1,102 @@
+pragma solidity ^0.5.11;
+pragma experimental ABIEncoderV2;
+
+import "openzeppelin-solidity/contracts/math/SafeMath.sol";
+
+
+/**
+ * @title Governor
+ * @dev The Governor holds the rights to stage and execute contract calls i.e. changing Livepeer protocol parameters. 
+ */
+contract Governor {
+
+    address public owner;
+
+    uint256 public nonce;
+
+    /// @dev mapping of updateHash (keccak256(update) => executeBlock (block.number + delay)
+    mapping(bytes32 => uint256) updates; 
+
+    using SafeMath for uint256;
+
+    event OwnershipTransferred(address indexed previousOwner, address indexed newOwner);
+
+    event UpdateStaged(Update update, uint256 delay);
+
+    event UpdateExecuted(Update update);
+
+    struct Update {
+        address[] target;
+        uint256[] value;
+        bytes[] data;
+        uint256 nonce;
+    }
+
+/**
```

Comment:
I see that the comment style here is different from the style used in the rest of the file. Can we be consistent with the comment style we use?

Reply:
fixed in cd04b83